cityofaustin/atd-moped
# [Bug] Cannot add a new advanced filter when the operator type is "blank/not blank"

## Associated issues

https://github.com/cityofaustin/atd-data-tech/issues/14302

## Testing
**URL to test:** 


https://deploy-preview-1178--atd-moped-main.netlify.app/moped/projects

**Steps to test:**

Open the advanced search dropdown
Pick a field that has "is blank" or "is not blank" as an option, such as Team, Partners or Inspector. 
Pick is blank or is not blank as an operator. You should be able to add another filter, in staging/prod right now you cannot. 



---
#### Ship list
